index.d.ts

```diff
@@ -0,0 +1,82 @@
+// This is manually built for now
+
+import { NamedNode, IndexedFormula, Fetcher, UpdateManager } from 'rdflib'
+
+declare const list: Array<PaneDefinition>
+declare const paneForIcon: { [key: string]: PaneDefinition }
+declare const paneForPredicate: {
+  [key: string]: {
+    pred: NamedNode
+    code: number
+  }
+}
+declare function register (
+  pane: PaneDefinition,
+  requireQueryButton?: boolean
+): void
+declare function byName (name: string): PaneDefinition | null
+
+/**
+ * All of the knowledge that a user accumulates throughout the current session
+ */
+export type DataBrowserSession = {
+  paneRegistry: PaneRegistry
+  store: IndexedFormula
+}
+
+/**
+ * The current context that a view is rendered into
+ */
+export type DataBrowserContext = {
+  dom: HTMLDocument
+  getOutliner: (dom: HTMLDocument) => any // @@ TODO Remove the use of getOutliner - only here as an interim until we have better solution
```

Comment:
Might want to prefer `unknown` over `any` here, so the caller will at least have to explicitly type the values they expect.